schema.json

```diff
@@ -0,0 +1,23 @@
+{
+    "title": "event",
+    "type": "object",
+    "properties": {
+ 	 "organizer": {
+        "name": "name",
+        "email": "email"
+  	 },
```

Comment:
For this part I think we can remove `"properties"` since it's easier if we just have "organizer" on the same level as "type", "title", etc.

Reply:
sounds good! I'll remove the "properties" tag